hack/start-flotta.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+
+usage()
+{
+cat << EOF
+Usage: $0 OPTIONS
+This script runs test for project-flotta to verify all flotta components can start running properly.
+OPTIONS:
+   -i      Image registry
+   -t      Target (ocp\k8s)
+   -o      Path to flotta-operator directory
+   -d      Path to flotta-device-worker directory
+   -y      Path to yggdrasil directory
+EOF
+}
+
+while getopts "i:t:o:d:y:" option; do
+    case "${option}"
+    in
+        i) IMG=${OPTARG};;
+        t) TARGET=${OPTARG};;
+        o) OPERATOR_PATH=${OPTARG};;
+        d) DEVICE_WORKER_PATH=${OPTARG};;
+        y) YGGDRASIL_PATH=${OPTARG};;
+        h)
+            usage
+            exit 0
+            ;;
+        *)
+            usage
+            exit 1
+            ;;
+    esac
+done
+
+if [[ -z $OPERATOR_PATH ]]; then
+    echo "Error: OPERATOR_PATH is required"
+    exit 1
+fi
+
+if [[ -z $DEVICE_WORKER_PATH ]]; then
+    echo "Error: DEVICE_WORKER_PATH is required"
+    exit 1
+fi
+
+if [[ -z $YGGDRASIL_PATH ]]; then
+    echo "Error: YGGDRASIL_PATH is required"
+    exit 1
+fi
+
+if [[ -z $IMG ]]; then
+    echo "Error: IMG is required"
+    exit 1
+fi
+
+if [[ -z $TARGET ]]; then
+    echo "Error: TARGET is required"
+    exit 1
+fi
+
+sh hack/start-operator.sh $IMG $TARGET $OPERATOR_PATH
+sh hack/start-agent.sh $TARGET $YGGDRASIL_PATH $DEVICE_WORKER_PATH
+
+echo start-flotta done!
```

Comment:
In addition can we avoid to print this if there were some errors in `  hack/start-operator.sh` and `hack/start-agent.sh ` (as in my previous comment for example)